### ISSUE TITLE
[Master-Merge ToDo] ICalls crash

### ISSUE THREAD

**Issue by chippmann** [utopia-rise/godot-kotlin#169]
Tested with `Texture.resourcePath` and `Curve.getPointCount()` which call `_icall_String` and `_icall_Long` respectively.

TestNode:
```kotlin
@RegisterClass
class TestingClass : Node() {
    @IntRange(0, 20, 2, Range.OR_GREATER)
    @RegisterProperty
    var rangeInt = 1

    @RegisterProperty
    lateinit var texture: Texture

    @RegisterProperty
    lateinit var curve: Curve

    @RegisterFunction
    override fun _ready() {
        println("Property: ${this::rangeInt.name} -> $rangeInt")
        println("Property: ${this::texture.name} -> TexturePath: ${texture.resourcePath}")
        println("Property: ${this::curve.name} -> ${curve.getPointCount()} Points")
    }
}
```

both properties are assigned in the `Inspector`

Stacktracke:
```
OpenGL ES 3.0 Renderer: AMD Radeon RX 5700 XT (NAVI10, DRM 3.35.0, 5.4.52-1-MANJARO, LLVM 10.0.0)
 
Property: rangeInt -> 12
handle_crash: Program crashed with signal 11
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /usr/lib/libc.so.6(+0x3c3e0) [0x7f504d5af3e0] (??:0)
-- END OF BACKTRACE --
```

**Comment by chippmann** [comment]
It still crashes, even though with a different error this time:
```
OpenGL ES 3.0 Renderer: AMD Radeon RX 5700 XT (NAVI10, DRM 3.35.0, 5.4.52-1-MANJARO, LLVM 10.0.0)
 
Running: /home/cedric/bin/Godot_v3.2.2-stable_x11.64 --path /home/cedric/projects/01_private/godot/03_libs/godot-kotlin/samples/mini-games --remote-debug 127.0.0.1:6007 --allow_focus_steal_pid 15473 --position 320,180 res://Node2D.tscn
Godot Engine v3.2.2.stable.official - https://godotengine.org
OpenGL ES 3.0 Renderer: AMD Radeon RX 5700 XT (NAVI10, DRM 3.35.0, 5.4.52-1-MANJARO, LLVM 10.0.0)
 
Uncaught Kotlin exception: kotlin.UninitializedPropertyAccessException: lateinit property ptr has not been initialized
        at  (0x7fc2f53180a7)
        at  (0x7fc2f5312835)
        at  (0x7fc2f5312535)
        at  (0x7fc2f5313215)
        at  (0x7fc2f534179d)
        at  (0x7fc2f5f11cf2)
        at  (0x7fc2f5f17391)
        at  (0x7fc2f5382853)
        at  (0x7fc2f6ace1f3)
        at  (0x7fc2f6ac80f2)
        at  (0x7fc2f536f4a3)
        at  (0x7fc2f536f6a6)
        at  (0x7fc2f5372a2d)
        at  (0x7fc2f536990b)
        at  (0x7fc2f5377b86)
        at  (0x7fc2f536f73d)
        at _ZN12NativeScript15instance_createEP6Object (0x2603309)
        at _ZN6Object10set_scriptERK6RefPtr (0xc59d88)
        at _ZN6Object3setERK10StringNameRK7VariantPb (0xc5a15b)
        at _ZNK10SceneState8instanceENS_12GenEditStateE (0x12e6f2f)
        at _ZNK11PackedScene8instanceENS_12GenEditStateE (0x12e869e)
        at _ZN4Main5startEv (0x2a5f315)
        at main (0x85829d)
        at __libc_start_main (0x7fc39f263002)
        at  (0x86695e)
        at  ((nil))
```

the example i tried:
```kotlin
@RegisterProperty
lateinit var spriteNodePath: NodePath

@RegisterProperty
lateinit var textureForSprite: Texture

@RegisterFunction
override fun _ready() {
    (getNode(spriteNodePath) as Sprite).texture = textureForSprite
}
```

**Comment by chippmann** [comment]
The issue is also present with the original example in this issue

**Comment by raniejade** [comment]
I fixed this #198, @chippmann can you verify that everything is working now?

**Comment by chippmann** [comment]
@raniejade Yeah i tested it when i revieved the PR. This cases described in this issue are working now as expected.
